src/Checkers/SourceCodeChecker.php

```diff
@@ -189,21 +189,24 @@ private function getMethodParameters(ReflectionMethod $method): array
      */
     private function getConstantNames(ReflectionClass $reflection): array
     {
-        return array_merge(...array_values((array_map(
-            function (ReflectionClassConstant $constant) use ($reflection): array {
-                foreach ($reflection->getTraits() as $trait) {
-                    if ($trait->hasConstant($constant->getName())) {
-                        return [];
+        return array_map(
+            fn (ReflectionClassConstant $constant): string => $constant->name,
+            array_filter(
+                $reflection->getReflectionConstants(),
+                function (ReflectionClassConstant $constant) use ($reflection): bool {
+                    if ($constant->class !== $reflection->name) {
+                        return false;
+                    }
+                    foreach ($reflection->getTraits() as $trait) {
```

Comment:
Maybe add a space here to space the conditions visually?

```suggestion

                    foreach ($reflection->getTraits() as $trait) {
```

Reply:
Sure!